src/validation.cairo

```diff
@@ -210,9 +210,15 @@ fn fee_and_merkle_root(block: @Block) -> Result<(u64, Hash), ByteArray> {
     let mut txids: Array<Hash> = array![];
     let mut total_fee = 0;
 
-    for tx in *block.txs {
+    // skipping the coinbase transaction
```

Comment:
Can you please move this comment to the `i != 0` line? Just to avoid possible confusion